packages/frontend/src/components/common/ResourceInfoPopup/ResourceInfoPopup.tsx

```diff
@@ -0,0 +1,54 @@
+import { HoverCard, Space, Text } from '@mantine/core';
+import { IconInfoCircle } from '@tabler/icons-react';
+import { FC } from 'react';
+import MantineIcon from '../MantineIcon';
+import { DashboardList } from './DashboardList';
+
+type Props = {
+    resourceUuid: string;
+    withChartData?: boolean;
+    description?: string;
+    projectUuid: string;
+};
+
+export const ResourceInfoPopup: FC<Props> = ({
+    resourceUuid,
+    description,
+    projectUuid,
+    withChartData = false,
+}) => {
+    return (
+        <HoverCard
```

Comment:
HoverCard!! nice, @joaoviana was actually trying to hack this using custom hook on top of popover or tooltip. we definitely missed we had a HoverCard component...

Reply:
I had to use it before 😄